### PR TITLE
Updated Spanish and add Catalan translations

### PR DESCRIPTION
**Pull request by Sappstal**

### CODE REVIEW

File: Nearby Sharing Windows/Resources/values-ca/strings.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+<string name="receive_missing_permissions">No es pot rebre sense els permisos necessaris!</string>
+<!-- #endregion -->
+	
+<!-- #region ReceiveActivity.cs -->
+<string name="share_file">Compartir arxiu</string>
+<string name="share_url">Compartir URL</string>
+
+<string name="send_missing_permissions">Falten alguns permisos necessaris!</string>
+<string name="wait_for_acceptance">Esperant reposta...</string>
+<string name="sending_template">Enviant ... {0}/{1} arxius... {2}%</string>
```
Comment (ShortDevelopment) [inline Nearby Sharing Windows/Resources/values-ca/strings.xml:94]: I changed this template
https://github.com/nearby-sharing/android/blob/6f84dfca6ad0fd0ce84d6d28d52d5a5bcc8cb9cc/src/Resources/values/strings.xml#L97

File: Nearby Sharing Windows/Resources/values-ca/strings.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+<string name="app_name">Ús compartit en proximitat (Nearby Sharing Windows)</string>
```
Comment (ShortDevelopment) [inline Nearby Sharing Windows/Resources/values-ca/strings.xml:3]: I removed "Windows" from the title

Comment (ShortDevelopment) [inline Nearby Sharing Windows/Resources/values-ca/strings.xml:3]: Please only use either the language-specific or english name

File: Nearby Sharing Windows/Assets/ca/MacAddressInfo.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿Si no pots rebre dades comprova la configuració, el registre de depuració i dona una ullada a les Preguntes Frecuents (en la Configuració).
+<br />
+Si et trobes amb un problema, per favor, informa directament a través de GitHub.
+<br />
+https://github.com/shortdevelopment/nearby-sharing-windows/issues
```
Comment (ShortDevelopment) [inline Nearby Sharing Windows/Assets/ca/MacAddressInfo.html:5]: The repo has moved to `https://github.com/nearby-sharing/android`

**Review by ShortDevelopment — CHANGES_REQUESTED** [review]

**Comment by Sappstal** [comment]
Hi! Sorry, I had some trouble with the branch and some files' older versions were duplicated. I'll close this PR and fix it.

File: src/Resources/values-ca/strings.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-<string name="app_name">Ús compartit en proximitat</string>
+<string name="app_name">Ús compartit en proximitat de Windows</string>
```
Comment (ShortDevelopment) [inline src/Resources/values-ca/strings.xml:3]:
```suggestion
<string name="app_name">Ús compartit en proximitat</string>
```

File: src/Resources/values-es/strings.xml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<string name="app_name">Uso compartido en proximidad</string>
+	<string name="app_name">Uso compartido en proximidad de Windows</string>
```
Comment (ShortDevelopment) [inline src/Resources/values-es/strings.xml:3]:
```suggestion
	<string name="app_name">Uso compartido en proximidad</string>
```

**Review by ShortDevelopment — CHANGES_REQUESTED** [review]

**Comment by ShortDevelopment** [comment]
@Sappstal No problem!
I recommend you delete your fork **AFTER** this is merged and create a fresh fork.
Then you can modify stuff by [creating a new branch](https://docs.github.com/en/get-started/exploring-projects-on-github/contributing-to-a-project#creating-a-branch-to-work-on) and open a PullRequests for your feature- / patch-branch.
It's also possible to create / edit a file [directly in the GitHub UI](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files)!

This way you wont have merge-conflicts all the time.

**Review by ShortDevelopment — CHANGES_REQUESTED** [review]
Last thing: Please add this entry to `./values-ca/strings.xml`:
```xml
<string name="assets_prefix">ca</string>
```

**Review by ShortDevelopment — APPROVED** [review]
Sry, everything is fine!
Thank you for your contribution!

**Pull request merged**